app/models/claim.rb

```diff
@@ -36,11 +36,11 @@ class Claim < ApplicationRecord
   validates :status, presence: true
   validates :reference, uniqueness: { case_sensitive: false }, allow_nil: true
 
-  scope :not_internal, -> { where.not(status: :internal) }
+  scope :not_internal_archived_or_discarded, -> { where.not(status: %i[internal archived discarded]) }
```

Comment:
You'd be better off having an `active` scope. Possibly worth moving these to a constant. Or even better, reverse the logic and only include the statuses with are active `ACTIVE_STATUSES = %i[draft submitted]`